clients/web/src/views/body/UsersView.js

```diff
@@ -34,9 +34,8 @@ var UsersView = View.extend({
     initialize: function (settings) {
         cancelRestRequests('fetch');
         this.collection = new UserCollection();
-        this.collection.on('g:changed', function () {
-            this.render();
-        }, this).fetch();
+        var promiseArray = [];
```

Comment:
In new JS code, use `let` for things that will be reassigned, and `const` otherwise. This should be `const`.